### PR TITLE
feat: Add calling pagination (SQCALL-295)

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/calling/Call.ts [hunk 1]
```diff
@@ -187,5 +194,14 @@ export class Call {
 
   removeParticipant(participant: Participant): void {
     this.participants.remove(participant);
+    this.updatePages();
+  }
+
+  updatePages() {
+    const newPages = chunk(this.participants(), NUMBER_OF_PARTICIPANTS_IN_ONE_PAGE) as Participant[][];
```
Comment (bennycode) [inline src/script/calling/Call.ts]:
```suggestion
    const newPages = chunk<Participant[][]>(this.participants(), NUMBER_OF_PARTICIPANTS_IN_ONE_PAGE);
```

File: src/script/calling/CallingRepository.ts [hunk 2]
```diff
@@ -1239,18 +1252,19 @@ export class CallingRepository {
...
     conversationId: ConversationId,
-    userId: UserId,
-    clientId: ClientId,
-    streams: MediaStream[],
+    remoteUserid: UserId,
+    remoteClientid: ClientId,
+    streams: readonly MediaStream[] | null,
   ): void => {
-    let participant = this.findParticipant(conversationId, userId, clientId);
+    let participant = this.findParticipant(conversationId, remoteUserid, remoteClientid);
```
Comment (bennycode) [inline src/script/calling/CallingRepository.ts]:
```suggestion
    let participant = this.findParticipant(conversationId, remoteUserId, remoteClientId);
```

File: src/script/calling/CallingRepository.ts [hunk 3]
```diff
@@ -1239,18 +1252,19 @@ export class CallingRepository {
 
   private readonly updateParticipantVideoStream = (
     conversationId: ConversationId,
-    userId: UserId,
-    clientId: ClientId,
-    streams: MediaStream[],
+    remoteUserid: UserId,
```
Comment (bennycode) [inline src/script/calling/CallingRepository.ts]:
```suggestion
    remoteUserId: UserId,
```

File: src/script/calling/CallingRepository.ts [hunk 4]
```diff
@@ -1239,18 +1252,19 @@ export class CallingRepository {
...
+    remoteClientid: ClientId,
```
Comment (bennycode) [inline src/script/calling/CallingRepository.ts]:
```suggestion
    remoteClientId: ClientId,
```

File: src/script/components/Icon.tsx [hunk 5]
```diff
@@ -38,6 +38,9 @@ const normalizeIconName = (name: string) =>
 const createSvgComponent = (svg: HTMLElement, displayName: string): React.FC<IconProps> => {
   const SVGComponent: React.FC<IconProps> = oProps => {
     const viewBox = svg.getAttribute('viewBox');
+    if (!viewBox) {
+      console.error('Svg icon must have a viewBox attribute');
```
Comment (bennycode) [inline src/script/components/Icon.tsx:42]: Is this log statement necessary? Can you throw an error to catch it in our code?

**Review by bennycode — CHANGES_REQUESTED** [review]

File: src/script/components/Icon.tsx (hunk 5 above)
Comment (thisisamir98) [reply src/script/components/Icon.tsx:42]: I talked about this with @AndyLnd 
Actually we get a react error but it's not that obviously demonstrating what's going wrong...

**Review by bennycode — APPROVED** [review]

**Pull request merged**